backend/src/main/java/com/hjjang/backend/domain/email/dto/Email.java

```diff
@@ -0,0 +1,44 @@
+package com.hjjang.backend.domain.email.dto;
+
+import java.util.Random;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
```

Comment:
```suggestion
@Getter
@Setter
```
Setter를 통해 Getter처럼 사용이 가능합니다.